Update blog about drop-in hours (#98)
2i2c-org/2i2c-brochure: content/blog/2022/product-community-lead-drop-in-notes/index.md

More important than anything is a deep interest in this ecosystem, and a desire to learn, but having prior experience with the Jupyter stack and similar workflows will be a big benefit.

## What level of technical expertise is needed for this position?

Ideally, they should have the expertise of a "power user" of the infrastructure and usecases that we work with.
We don't expect this role to have software development or cloud infrastructure engineering expertise, but they should understand what it is like to **use** these tools from a user's perspective, and have an intuition about how to improve the workflows.
They should be able to communicate with and collaborate with engineers effectively, so understanding development workflows will be helpful, but they do not need core expertise in software development.

## Is it kind of like a scientific / technical writer?

The skills of a scientific or technical writer would certainly be relevant for this position.
For example, somebody that enjoyed technical documentation or scientific communication likely has one relevant skillset for this role.

> However, the rolle will do much more than _just_ technical writing - they will also create and execute on organizational strategy, collabate with our team around designing and planning our work, etc.

Author: ```suggestion
However, the role will do much more than _just_ technical writing - they will also create and execute on organizational strategy, collaborate with our team around designing and planning our work, etc.
```

> ## Is the scientific knowledge important?

Author: ```suggestion
## Is scientific knowledge in a specific domain important?
```

We don't expect this person to have deep knowledge in any particular scientific field, though experience with scientific and educational workflows in general will be very beneficial.

> 2i2c's goal is to biuld expertise in the aspects of interactive computing workflows that overlap between many rearch / educational communities.

Author: ```suggestion
2i2c's goal is to build expertise in the aspects of interactive computing workflows that overlap between many research / educational communities.
```

We hope to work with leaders from these communities to leverage their domain expertise, and to empower them to lead others.
Similar to having technical expertise, the important thing is having an intuition for what it's like to use the infrastructure, what _kinds of problems_ our target communities are trying to solve, and how to learn more about their perspectives.

## Does the role involve writing grants for the sustainability of the project?

This role isn't tasked specifically with identifying and writing grants, though it is likely that they will collaborate with others at 2i2c in crafting grant language or pursuing similar opportunities.
2i2c is a small organization and so we all tend to wear many hats, though we try to keep each person's efforts focused on their core responsibilities.

> In this case, those responsibilities are community guidance and engagement, as well as service and product guidance and design in collaboration with our team.

Author: ```suggestion
In this case, those responsibilities are community guidance and engagement, as well as service / product design in collaboration with our team.
```

> ## Community Role - what kinds of communications should it oversee?

Author: ```suggestion
## What kinds of communications would this role oversee?
```

This is hard to know right now and may evolve over time, but we expect this role will primarily communicate with pre-existing communities that we serve.
They will also foster communication between leaders within these communities.
They may interface with communities that 2i2c seeks to serve in the future (for example, in collaboration with sales and partnerships), but their focus will be on cultivating and guiding the communities we already work with.

## Term for this position

We hire all team members on one-year contracts with an annual review and the expectation to renew the contract for another year (unless we explicitly say otherwise).

> ## Where are the communities you work with? All ove the world? What kinds of people?

Author: ```suggestion
## Where are the communities you work with?
```

> Here are a few characteristics of the communities we work with:

Author: ```suggestion
Here are a few characteristics of the communities we _currently_ work with, though we aspire to broaden this group significantly (for example, we wish to expand our focus outside of North American and Europe):
```

- They tend to be focused in North America and in Europe (this is likely due to our pre-existing connections)
- They cover a broad range of use-cases, from small- and large-scale education in single institutions, to small- and large-scale research communities that are distributed across the globe.
- A common pattern is to work with "community champions" that are interested in our technology and serve as leaders to bring these workflows to their respective communities. We seek to build more connections with people like this, and cultivate our relationship with them.

Aspirationally, we hope to grow our reach to more communities outside of these countries, in order to achieve our mission of reaching a global audience.
Doing so will require more sophistication in our coordination and pricing, in order to match our services and support model with communities outside of North America and Europe.

> ## Not much structure right now. How do you plan to add more structure over time? What support will this role have?

Author: ```suggestion
## 2i2c is young and without much structure. How do you plan to add more structure over time? What support will this role have?
```
